cfspopcon/formulas/energy_confinement/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Interface to different energy confinement scalings and routines to calculate the plasma stored energy."""
+
+from . import plasma_stored_energy, read_energy_confinement_scalings, solve_for_input_power
+
+__all__ = ["plasma_stored_energy", "read_energy_confinement_scalings", "solve_for_input_power"]
```

Comment:
I think we should export the actual functions not the modules they are defined in, right?

Reply:
I've changed it so that we import the functions into the submodules of formulas. However, we still only import the modules into formulas. So, you'd need
```
from cfspopcon.formulas.energy_confinement import calc_plasma_stored_energy
```
while  `from cfspopcon.formulas import calc_plasma_stored_energy` won't work. I prefer this to remind a user where functions are actually being defined, but can import all functions into the main `formulas` module if you'd like. What do you think?